src/smtsolvers/LookaheadSMTSolver.cc

```diff
@@ -42,17 +44,17 @@ lbool LookaheadSMTSolver::solve_() {
         }
     }
     switch (res) {
-        case LALoopRes::unknown_final:
-            return l_Undef;
-        case LALoopRes::sat:
-            return l_True;
-        case LALoopRes::unsat: {
-            ok = false;
-            return l_False;
-        }
-        default:
-            assert(false);
-            return l_Undef;
+    case LALoopRes::unknown_final:
```

Comment:
The new indentation seems wrong. I think the previous one is correct.

Reply:
Changed